octobot_commons/enums.py

```diff
@@ -74,3 +83,17 @@ class OctoBotTypes(Enum):
     BINARY = "binary"
     PYTHON = "python"
     DOCKER = "docker"
+
+
+class MarkdownFormat(Enum):
+    ITALIC = "_"
+    BOLD = "**"
+    CODE = "`"
+    IGNORE = ""
+    NONE = ""
+
+
+class NotificationCategory(Enum):
```

Comment:
Why this is required to be in OctoBot-Commons ?

Reply:
moved to OctoBot-Notifications